flags.md

```diff
@@ -0,0 +1,141 @@
+--define:DEV                          # tags: dev
+--listCmd                             # tags: dev
+
+--define:DEBUG                        # tags: debug
+--debugger:on                         # tags: debug
+--debuginfo                           # tags: debug
+--linedir:on                          # tags: debug
+
+--define:PROFILE                      # tags: memprofile
+--profiler:off                        # tags: memprofile
+--stackTrace:on                       # tags: memprofile
+--define:memProfiler                  # tags: memprofile
+
+--define:PROFILE                      # tags: profiler
+--profiler:on                         # tags: profiler
+--stackTrace:on                       # tags: profiler
+
+--debugger:native                     # tags: profilenative
+
+# Compilation
+
+--define:OPTIMIZED                    # tags: optimized
+
+--define:strip                        # tags: release
+--passC:-flto                         # tags: release, unix
+--passL:-flto                         # tags: release, unix
+
+# Compilation: full
+
+--define:ssl                          # tags: full
+
+# Compilation: mini/web
+
+--define:SAFE                         # tags: safe
+--define:MINI                         # tags: mini
+--define:WEB                          # tags: web
+
+--verbosity: 3                        # tags: web
+
+--define:NOASCIIDECODE                # tags: noasciidecode
+--define:NOCLIPBOARD                  # tags: noclipboard
+--define:NODIALOGS                    # tags: nodialogs
+--define:NOERRORLINES                 # tags: noerrorlines
+--define:NOGMP                        # tags: nogmp
+--define:NOPARSERS                    # tags: noparsers
+--define:NOSQLITE                     # tags: nosqlite
+--define:NOWEBVIEW                    # tags: nowebview
+
+# Compilation: defaults
+--verbosity:1                         # tags: default
+--hints:on                            # tags: default
+--hint:ProcessingStmt:off             # tags: default
+--hint:XCannotRaiseY:off              # tags: default
+--warning:GcUnsafe:off                # tags: default
+--warning:CastSizes:off               # tags: default
+--warning:ProveInit:off               # tags: default
+--warning:ProveField:off              # tags: default
+--warning:Uninit:off                  # tags: default
+--warning:BareExcept:off              # tags: default
+--threads:off                         # tags: default
+--skipUserCfg:on                      # tags: default
+--colors:off                          # tags: default
+--define:danger                       # tags: default
+--panics:off                          # tags: default
+--mm:orc                              # tags: default
+--define:useMalloc                    # tags: default            
+--checks:off                          # tags: default
+--cincludes:extras                    # tags: default
+--opt:speed                           # tags: default
```

Comment:
Question:
* Should speed be the default, or should this be used just for releases?
releases are for `user`, `bench` or `ci`.

If `--opt:speed` increases the compilation time, I think this would be a good idea to disable this for `--who dev`.

Reply:
@drkameleon, what do you think?